e2e/cypress/integration/02_global_filter.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('global projects filter', () => {
+  before(() => {
+    cy.login('/').then(() => {
+      let admin = JSON.parse(localStorage.getItem('chef-automate-user'))
+      cy.request({
+          auth: { bearer: admin.id_token },
+          method: 'POST',
+          url: `/apis/iam/v2beta/projects`,
```

Comment:
[tiniest nit] no need for backticks, as there's no string interpolation happening here; let's stick with single or double quotes